main/res/values/strings.xml

```diff
@@ -554,10 +555,17 @@
     <string name="cache_filter_distance_use_current_position">Use current position</string>
 
     <string name="cache_filter_status_select_all">All</string>
-    <string name="cache_filter_status_select_only_found_yes">Only Found</string>
-    <string name="cache_filter_status_select_only_found_no">Only Not Found</string>
-    <string name="cache_filter_status_select_only_own_yes">Only Owned</string>
-    <string name="cache_filter_status_select_only_own_no">Only Not Owned</string>
+    <string name="cache_filter_status_select_yes">Yes</string>
+    <string name="cache_filter_status_select_no">No</string>
+
+    <string name="cache_filter_status_select_label_found">Found</string>
+    <string name="cache_filter_status_select_label_owned">Owned</string>
+    <string name="cache_filter_status_select_label_stored">Stored</string>
+    <string name="cache_filter_status_select_label_favorite">Favored</string>
+    <string name="cache_filter_status_select_label_watchlist">On Watchlist</string>
+    <string name="cache_filter_status_select_label_premium">Premium</string>
+    <string name="cache_filter_status_select_label_solved_mystery">Solved Mystery</string>
+    <string name="cache_filter_status_select_infotext_solved_mystery">This filter setting will only affect mystery caches.\n\n A mystery is considered \'solved\' if it has either changed coordinates (gc.com only) or a valid final waypoint filled with coordinates.</string>
```

Comment:
we should write the domain name / service name as `geocaching.com` - `gc.com` is something different (although we use it frequently as abbreviation)

Reply:
Very true. I have removed the "like gc.com" phrase